org.eclipse.xtext.xtext.ui/src/org/eclipse/xtext/xtext/ui/wizard/project/AdvancedNewProjectPage.xtend

```diff
@@ -81,8 +84,25 @@ class AdvancedNewProjectPage extends WizardPage {
 					it.InfoDecoration(AdvancedNewProjectPage_projIde_description)
 					layoutData = new GridData(SWT.LEFT, SWT.CENTER, true, false)
 				]
-				createTestProject = CheckBox [
-					text = Messages.WizardNewXtextProjectCreationPage_TestingSupport
+				new Composite(it, SWT.NONE) => [
+					val _layout = new GridLayout(3, false)
```

Comment:
_underscore? maybe rename it to gridLayout or use an initializer with `layout = new GridLayout(3, false) => [ marginWith ... ]`

Reply:
initializer is a good idea